pkg/database/vercode.go

```diff
@@ -204,7 +224,8 @@ func (r *Realm) FindVerificationCode(db *Database, code string) (*VerificationCo
 
 	var vc VerificationCode
 	if err := db.db.
-		Where("realm_id = ? AND (code IN (?) OR long_code IN (?))", r.ID, hmacedCodes, hmacedCodes).
+		Joins("JOIN realms ON realms.id = verification_codes.realm_id").
```

Comment:
per the earlier comment - not loading the realm relation here can avoid a join.

Reply:
Done